### PR TITLE
adding documentation

### PR DESCRIPTION
**Pull request by evakule**

### CODE REVIEW

File: src/main/java/com/somn/controller/AccountController.java [hunk 1]
```diff
@@ -29,10 +35,24 @@
 
 @RestController
 @RequestMapping(value = "api/v1/accounts")
+@Api(value = "Set of endpoints for Creating, Retrieving, "
+    + "Updating and Deleting of Accounts",
+    produces = MediaType.APPLICATION_JSON_VALUE)
 public class AccountController {
+  private static final int OK = 200;
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: You can reuse org.springframework.http.HttpStatus enum

File: src/main/java/com/somn/controller/AccountController.java [hunk 2]
```diff
@@ -29,10 +35,24 @@
...
+  private static final int BAD_REQUEST = 400;
+  private static final int CREATED = 201;
+  private static final int NO_CONTENT = 204;
+  
   @Autowired
   private AccountEntityService accountEntityService;
   
+  @ApiOperation("Shows all customers accounts except their balance")
+  @ApiResponses(value = {
+      @ApiResponse(code = OK, message = "OK"),
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: Fulfil messages with valuable information

File: src/main/java/com/somn/controller/AccountController.java [hunk 3]
```diff
@@ -45,6 +65,11 @@
     }
   }
   
+  @ApiOperation("Showss single customers account except balance")
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: fix typos please

File: src/main/java/com/somn/dto/CustomerAccountDTO.java [hunk 4]
```diff
@@ -1,11 +1,20 @@
 package com.somn.dto;
 
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+@ApiModel(
+    value = "Account",
+    description = "Class that represents an account")
```
Comment (ssuchkov) [inline src/main/java/com/somn/dto/CustomerAccountDTO.java]: Business is not really care about classes

File: src/main/java/com/somn/controller/AccountController.java (hunk 2 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Ok

File: src/main/java/com/somn/controller/AccountController.java (hunk 1 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Impossible to do this

File: src/main/java/com/somn/controller/AccountController.java (hunk 3 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Ok

File: src/main/java/com/somn/dto/CustomerAccountDTO.java (hunk 4 above)
Comment (evakule) [reply src/main/java/com/somn/dto/CustomerAccountDTO.java]: Ok

File: src/main/java/com/somn/controller/AccountController.java [hunk 5]
```diff
@@ -29,10 +36,21 @@
...
+  
   @Autowired
   private AccountEntityService accountEntityService;
   
+  @ApiOperation("Display all accounts in the system. Used only by accountant.")
+  @ApiResponses(value = {
+      @ApiResponse(code = ResponseCode.OK, message =
+          "All accounts selected successfully."),
+      @ApiResponse(code = ResponseCode.NOT_FOUND, message =
+          "There is no accounts associated with any user.")
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: are

File: src/main/java/com/somn/controller/AccountController.java [hunk 6]
```diff
@@ -93,6 +134,15 @@
     }
   }
   
+  @ApiOperation("Withdraw money from account. Used by customer.")
+  @ApiResponses(value = {
+      @ApiResponse(code = ResponseCode.OK, message =
+          "Transaction successful"),
+      @ApiResponse(code = ResponseCode.BAD_REQUEST, message =
+          "Couldn't make transaction. Make sure that operation value "
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: We cannot refer to existing hardcoded or configured values

File: src/main/java/com/somn/controller/CustomerController.java [hunk 7]
```diff
@@ -20,10 +27,22 @@
...
   @Autowired
   private CustomerEntityService customerEntityService;
   
+  @ApiOperation("Display all customers in the system except their balances. "
+      + "Used only by admin.")
+  @ApiResponses(value = {
+      @ApiResponse(code = ResponseCode.OK, message =
+          "All customers selected successfully."),
+      @ApiResponse(code = ResponseCode.NOT_FOUND, message =
+          "There is no customers in system. Try to add some customer")
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/CustomerController.java]: are

File: src/main/java/com/somn/documentation/SwaggerConfig.java [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+@EnableSwagger2
+@PropertySource("classpath:swagger.properties")
+@ComponentScan(basePackages = "com.somn.controller")
+@Configuration
+public class SwaggerConfig {
+  
+  private static final String SWAGGER_API_VERSION = "1.0";
+  private static final String LICENSE_TEXT = "License";
+  private static final String title = "Online Bank REST API";
+  private static final String description = "RESTful API for Online Bank";
```
Comment (ssuchkov) [inline src/main/java/com/somn/documentation/SwaggerConfig.java]: is that up to naming conventions?

Comment (evakule) [reply src/main/java/com/somn/documentation/SwaggerConfig.java]: Ok

File: src/main/java/com/somn/controller/CustomerController.java (hunk 7 above)
Comment (evakule) [reply src/main/java/com/somn/controller/CustomerController.java]: Ok

File: src/main/java/com/somn/controller/AccountController.java (hunk 6 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Ok

File: src/main/java/com/somn/controller/AccountController.java (hunk 5 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Ok

**Pull request merged**